src/modules/SdnDiag.Server/public/Set-SdnVMNetworkAdapterPortProfile.ps1

```diff
@@ -7,9 +7,11 @@ function Set-SdnVMNetworkAdapterPortProfile {
     .PARAMETER MacAddress
         Specifies the MAC address of the VM network adapter.
     .PARAMETER ProfileId
-        The InstanceID of the Network Interface taken from Network Controller.
+        The InstanceID of the Network Interface taken from Network Controller. If ommited, defaults to an empty GUID to enable network connectivity for non-NC managed VMs.
```

Comment:
The changes all looks good. There was a question raised when engineer start to use this set port profile. We have only one example with no ProfileId provided. Can we leverage this PR to add one more example to demo how to update ProfileId which is the most common usage for this.

Reply:
So this was new to me as well until recently but comes into play when you deploy a VM outside of WAC on a Hyper-V host where VFP has been enabled. In this scenario even though the VM is not going to be managed by SDN, we need to have an empty guid defined to allow communication to work.

See https://learn.microsoft.com/en-us/azure-stack/hci/manage/manage-default-network-access-policies-virtual-machines#vms-created-outside-of-windows-admin-center for more details.

Note: I have a PR already submitted to docs team to update the article to leverage SdnDiagnostics module to consolidate the tools out in the wild into a source repository.